package.json

```diff
@@ -2,7 +2,7 @@
   "name": "matroid",
   "version": "1.0.1",
```

Comment:
Let's update the version! I'm tempted to call it `2.0.0` since it's such a big expansion, but there are no breaking changes, so let's do `1.1.0`.

Reply:
Fixed!